mindsdb/integrations/handlers/huggingface_inference_api_handler/huggingface_inference_api_handler.py

```diff
@@ -0,0 +1,81 @@
+import os.path
+from typing import Optional, Dict
+
+import json
+import pandas as pd
+import requests
+
+from .config_parser import ConfigParser
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class HuggingFaceInferenceAPIHandler(BaseMLEngine):
+    """
+    Integration with the Hugging Face Inference API.
+    """
+
+    name = 'huggingface_inference_api'
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
+        if 'using' not in args:
+            raise Exception("Hugging Face Inference engine requires a USING clause! Refer to its documentation for more details.")
+
+        config = ConfigParser(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config.yaml'))
```

Comment:
Shouldn't the user be able to pass these params via `USING`? In particular, the model name seems quite important.

Reply:
Yes, the users can pass a model of their choice now. If it is not provided, the recommend model will be used.